src/Symfony/Component/DependencyInjection/Attribute/AutowireInline.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Attribute;
+
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+
+/**
+ * Allows inline service definition at the class level.
```

Comment:
this description seems confusing as the attribute is not applied at class level.

Reply:
Done [here](https://github.com/symfony/symfony/compare/c94322321acafdbaf32044e58b213223645497ce..0ad67afe94ac26fd51bb7d4a67792721c50ed9de)